README.md

```diff
@@ -1,32 +1,291 @@
-# Stakewise python realization of `@openzeppelin/merkle-tree`
+# Stakewise python fork of [@openzeppelin/merkle-tree](https://github.com/OpenZeppelin/merkle-tree)
 
-## NB! Library is not well tested and not ready for the production use
-
-**A Python library to generate merkle trees and merkle proofs.**
-Well suited for airdrops and similar mechanisms in combination with OpenZeppelin Contracts [`MerkleProof`] utilities.
+A Python library to generate merkle trees and merkle proofs.
```

Comment:
"Merkle" should start with capital M when used in a plain text (not in code). Fix here and below.